src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -20,6 +20,10 @@ public class GlobbingUrlBuilder
     {
         private static readonly char[] PatternSeparator = new[] { ',' };
 
+        // Valid whitespace characters defined by the HTML5 spec.
+        private static readonly char[] ValidAttributeWhitespaceChars =
+            new[] { '\t', '\n', '\u000C', '\r', ' ' };
```

Comment:
@IDisposable yeah; could e.g. be `public static readonly` in `UrlResolutionTagHelper`. We don't have many *Constants classes and a reference from here to `UrlResolutionTagHelper` isn't __too__ ugly.